drivers/flash/flash_stm32_qspi.c

```diff
@@ -206,12 +264,76 @@ static int qspi_write_access(const struct device *dev, QSPI_CommandTypeDef *cmd,
 	return dev_data->cmd_status;
 }
 
+static int qspi_write_enable(const struct device *dev)
+{
+#ifndef CONFIG_FLASH_STM32_OSPI_MIMIC_QSPI
+	QSPI_CommandTypeDef cmd_write_en = {
```

Comment:
Any reason for not using HAL_QSPI_COMMAND, as I see a nice define at the top, making it possible to get rid of the #ifndef construction. But this also requires adding an additional define for QSPI_INSTRUCTION_1_LINE and HAL_OSPI_INSTRUCTION_1_LINE

Reply:
As mentioned in previous comment, this will get too complex to the other side, because the `#define` will become quite large as it needs to incorporate QSPI/OSPI and single/four line output. Therefore keeping as is.